AQL: Add REPLACE_NTH() array function (#319)
arangodb/docs: 3.7/aql/functions-array.md

{% endaqlexample %}
{% include aqlexample.html id=examplevar type=type query=query bind=bind result=result %}

## REPLACE_NTH()

> `REPLACE_NTH(anyArray, position, replaceValue, defaultPaddValue) → newArray`

Reviewer: ```suggestion
`REPLACE_NTH(anyArray, position, replaceValue, defaultPaddingValue) → newArray`
```

> Replace the element at *position* from the *anyArray* with *replaceValue*. If *defaultPaddValue* is specified, and *position* points after an already existing entry in *anyArray*, *defaultPaddValue* is inserted as many times as needed to place *replaceValue* at the end of *annyArray*

Reviewer: ```suggestion
Replace the element at *position* from the *anyArray* with *replaceValue*. If *defaultPaddingValue* is specified, and *position* points behind an already existing entry in *anyArray*, *defaultPaddingValue* is inserted as many times as needed to place *replaceValue* at the end of *anyArray*
```

- **anyArray** (array): array with elements of arbitrary type
- **position** (number): the position of the element to replace. Positions start
  at 0. Negative positions are unsupported.

>   If *position* is out of bounds, and *defaultPaddValue* is not specified, 

Reviewer: ```suggestion
  If *position* is out of bounds, and *defaultPaddingValue* is not specified, 
```
